apps/speed.c

```diff
@@ -84,6 +84,10 @@
 # define RSA_DEFAULT_PRIME_NUM 2
 #endif
 
+#ifndef SIZE_MAX
+#include <internal/numbers.h>
+#endif
```

Comment:
I would not make this conditional on with `SIZE_MAX` is defined or not. Just include the header always. The header itself is sufficiently clever to figure out whether SIZE_MAX is already defined. Also I would move this include up to where all the other includes are. Finally internal headers should be in double quotes, i.e. `#include "internal/numbers.h"`